### ISSUE TITLE
attribute error at reference_astrometry stage in vizier tools

### ISSUE THREAD

**Issue by mpgh** [pyDANDIA/pyDANDIA#99]
Related to the availability of the vizier service, the following bug may occur at the recerence_astrometry stage. Astropy version 3.2. was used for that.

Traceback (most recent call last):
  File "/pyDANDIA/pyDANDIA/run_stage.py", line 197, in <module>
    run_stage_stand_alone()
  File "/pyDANDIA/pyDANDIA/run_stage.py", line 69, in run_stage_stand_alone
    dy=params['dy'])
  File "/anaconda3/lib/python3.7/site-packages/pyDANDIA-0.1-py3.7-linux-x86_64.egg/pyDANDIA/reference_astrometry.py", line 90, in run_reference_astrometry
    stellar_density_threshold)
  File "/anaconda3/lib/python3.7/site-packages/pyDANDIA-0.1-py3.7-linux-x86_64.egg/pyDANDIA/reference_astrometry.py", line 291, in catalog_objects_in_reference_image
    image_wcs,log,'Gaia')
  File "/anaconda3/lib/python3.7/site-packages/pyDANDIA-0.1-py3.7-linux-x86_64.egg/pyDANDIA/wcs.py", line 153, in fetch_catalog_sources_for_field
    radius)
  File "/anaconda3/lib/python3.7/site-packages/pyDANDIA-0.1-py3.7-linux-x86_64.egg/pyDANDIA/vizier_tools.py", line 66, in search_vizier_for_gaia_sources
    qs = Gaia.cone_search_async(c, r)
  File "/anaconda3/lib/python3.7/site-packages/astroquery/gaia/core.py", line 446, in cone_search_async
    dump_to_file=dump_to_file)
  File "/anaconda3/lib/python3.7/site-packages/astroquery/gaia/core.py", line 367, in __cone_search
    background=background)
  File "/anaconda3/lib/python3.7/site-packages/astroquery/utils/tap/core.py", line 318, in launch_job_async
    job.set_failed(True)
AttributeError: 'Job' object has no attribute 'set_failed'

**Comment by rachel3834** [comment]
Looks like this may be a version issue in the package astroquery.  I've tested this using astroquery v0.3.10 and the latest version (0.4.1) and I've not been able to re-produce this error.  I think it is internal to astroquery itself (a modified attribute), so I've added some exception handling to give an more meaningful error message but aside from that there isn't much I can do.  It may be related to a local bandwidth issue.
